modules/file-extension-common/pom.xml

```diff
@@ -10,13 +10,14 @@
     <groupId>org.mule.modules</groupId>
     <artifactId>mule-module-file-extension-common</artifactId>
     <name>Mule File Extension Common</name>
-    <packaging>jar</packaging>
+    <packaging>mule-plugin</packaging>
     <description>
         Reusable components for extensions built using Mule's SDK which operate over file systems (e.g: file, ftp, sftp, etc)
     </description>
 
     <properties>
         <formatterConfigPath>../../formatter.xml</formatterConfigPath>
+        <mule.plugins.maven.app.plugins.version>1.0.0-SNAPSHOT</mule.plugins.maven.app.plugins.version>
```

Comment:
Yes, an internal version was created but master branch on this plugin is still 1.0.0-SNAPSHOT.